### ISSUE TITLE
Need docs for ARIA tabpanel and tablist roles

### ISSUE THREAD

**Issue by rg-galieva** [mdn/sprints#1750]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

These pages are missing: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tablist_Role (404 error)
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tabpanel_Role (404 error)

References from the page are unavailable: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role
